pkg/kubelet/events/event.go

```diff
@@ -67,6 +67,7 @@ const (
 	FailedNodeAllocatableEnforcement     = "FailedNodeAllocatableEnforcement"
 	SuccessfulNodeAllocatableEnforcement = "NodeAllocatableEnforced"
 	UnsupportedMountOption               = "UnsupportedMountOption"
+	SandBoxChanged                       = "SandboxChanged"
```

Comment:
Please make the name of the constant match the event type string for consistency.